components/financial_assistance/app/views/financial_assistance/applications/_review_esi_benefit.html.erb

```diff
@@ -1,94 +1,85 @@
-<% if @bs4 %>
-  <% review_benefits_esi_hash(benefit).each do |key, value| %>
-    <tr>
-      <td class="col-6"><%= key %></td>
-      <td class="col-6"><%= value %></td>
-    </tr>
-  <% end %>
-<% else %>
-  <div class="border_bottom border-sides form-content">
+<div class="border_bottom border-sides form-content">
+  <div class="row ptb no-buffer row-height">
+    <div class="col-md-6">Employer Name</div>
+    <div class="col-md-6"><%= benefit.employer_name %></div>
+  </div>
+  <% if !FinancialAssistanceRegistry.feature_enabled?(:disable_employer_address_fields) %>
     <div class="row ptb no-buffer row-height">
-      <div class="col-md-6">Employer Name</div>
-      <div class="col-md-6"><%= benefit.employer_name %></div>
+      <div class="col-md-6">EMPLOYER ADDRESSS LINE 1</div>
+      <div class="col-md-6"><%= benefit.employer_address.address_1 %></div>
     </div>
-    <% if !FinancialAssistanceRegistry.feature_enabled?(:disable_employer_address_fields) %>
+
+    <% if benefit.employer_address.address_2.present? %>
       <div class="row ptb no-buffer row-height">
-        <div class="col-md-6">EMPLOYER ADDRESSS LINE 1</div>
-        <div class="col-md-6"><%= benefit.employer_address.address_1 %></div>
+        <div class="col-md-6">EMPLOYER ADDRESSS LINE 2</div>
+        <div class="col-md-6"><%= benefit.employer_address.address_2 %></div>
       </div>
+    <% end %>
 
-      <% if benefit.employer_address.address_2.present? %>
-        <div class="row ptb no-buffer row-height">
-          <div class="col-md-6">EMPLOYER ADDRESSS LINE 2</div>
-          <div class="col-md-6"><%= benefit.employer_address.address_2 %></div>
-        </div>
-      <% end %>
+    <div class="row ptb no-buffer row-height">
+      <div class="col-md-6">CITY</div>
```

Comment:
Looks like there might be a few places in this view that need translations?

Reply:
this is a legacy code

this partial was originally updated for bs4 some months ago, but it is no longer being used when bs4 is on